src/Components/CreateWorkflow/CreateWorkflow.tsx

```diff
@@ -9,18 +11,22 @@ import { serviceUrl, resolver } from "Config/servicesConfig";
 import queryString from "query-string";
 import { Add32 } from "@carbon/icons-react";
 import { FlowTeam, ComposedModalChildProps, ModalTriggerProps, WorkflowExport, CreateWorkflowSummary } from "Types";
+import { FeatureFlag } from "Config/appConfig";
 import styles from "./createWorkflow.module.scss";
 
 const workflowDagEngine = new WorkflowDagEngine({ dag: null });
 
 interface CreateWorkflowProps {
-  team: FlowTeam;
-  teams: FlowTeam[];
+  isSystem: boolean;
+  team: FlowTeam | null;
+  teams: FlowTeam[] | null;
   hasReachedWorkflowLimit: boolean;
 }
 
-const CreateWorkflow: React.FC<CreateWorkflowProps> = ({ team, teams, hasReachedWorkflowLimit }) => {
+const CreateWorkflow: React.FC<CreateWorkflowProps> = ({ isSystem, team, teams, hasReachedWorkflowLimit }) => {
```

Comment:
An aside that I saw the other day was recommending against using the `React.FC<>` format and just using the interface as the type itself. If needing to use `children` then add it a prop with type of `React.ReactNode`

[A blog post about it](https://fettblog.eu/typescript-react-why-i-dont-use-react-fc/)